helm-go-lib/main.go

```diff
@@ -174,6 +176,63 @@ func uninstallRelease(namespace string, kubeToken string, apiServer string, caFi
 	return C.CString("ok")
 }
 
+//export upgradeChart
+func upgradeChart(namespace string, kubeToken string, apiServer string, caFile string, releaseName string, chartName string, overrideValues string) *C.char {
+	actionConfig, err := buildActionConfig(namespace, kubeToken, apiServer, caFile)
+	if err != nil {
+		log.Printf("%+v\n", err)
+		return C.CString(err.Error())
+	}
+
+	client := action.NewUpgrade(actionConfig)
+	client.Namespace = namespace
+	client.CleanupOnFail = true
+
```

Comment:
I think we also want to specify `client.ResetValues = true`:
```
    --reset-values                               when upgrading, reset the values to the ones built into the chart
```

Reply:
Oh duh, it is even in the design doc 🙈 sorry!